index.html

```diff
@@ -3,23 +3,42 @@
 <head>
   <link rel="stylesheet" href="base.css">
   <link rel="stylesheet" href="module.css">
-  <!--Need to link all three CSS files?-->
-  <!-- Correct place for link tags? -->
+  <link rel="stylesheet" href="layout.css">
+  <script src="marked.js"></script>
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.1.1/jquery.min.js"></script>
   <!-- Need rel attribute? -->
+  <!-- /*<style>a{text-decoration:none}</style>*/ -->
+  <!-- Use above line of code or use line in module.css to remove link underline?-->
 </head>
 <body>
   <header>
+    <nav class="main-nav">
+      <ul>
+        <li class="nav-item" data-content="projects"><a href="#">My Projects</a></li>
+        <li class="nav-item" data-content="aboutMe"><a href="#">About Me</a></li>
```

Comment:
Excellent work using data attributes to identify the different list items/links in your nav!